### PR TITLE
#167727440 Setup Sequelize

### PR DESCRIPTION
**Pull request by O-Osahon**
### What does this PR do?
- Sets up the application to use Sequelize

### Description of tasks to be completed
- install `sequelize-cli`
- add `sequelize` and `pg-hstore` the application's dependencies
- Add  a `.sequelizerc` file, instructing how Sequelize should be bootstrapped
- Set database credentials in the `.env` file 
- Add migration and "undo migration" scripts to `package.json`

### How should this be manually tested?
-  Clone or download this repository
-  On your terminal, `cd` into the application's directory and run: `git checkout ch-setup-sequelize-167727440`
-  Set the following environment variables in your `.env` file:
```
DEV_DB_USER // database username
DEV_DB_PASSWORD // database password
DEV_DATABASE  //name of your local database
DATABASE_URL  // production database url
```

- If you have not, install [Postgresql](https://www.postgresql.org/download)
- Run `npm run db:migrate`
- Check your database, you should now have both Users and Departments table.
- To undo migration, run `npm run undo:migrate`

### What are the relevant Pivotal Tacker stories?
[#167727440](https://www.pivotaltracker.com/story/show/167727440)

### CODE REVIEW

**Review by parkerthegeniuschild — CHANGES_REQUESTED** [review]
I can confirm that the migration (create and rollback) functionality works.
Proof below:

image

Meanwhile, could you quickly fix your Hound errors?

File: src/config/passport.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import passport from 'passport'
-import passportLocal from "passport-local";
+import passport from 'passport';
```
[the remark this answers is not in the document]
Comment (parkerthegeniuschild) [reply src/config/passport.js:1]: You can fix this by adding `{
    "parserOptions": {
        "sourceType": "module",
    }
}` to the `.eslintrc.json` config file.

And you'll have 100% green checks for Hound.

File: src/config/db.config.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const dotenv = require('dotenv');
```
[the remark this answers is not in the document]
Comment (parkerthegeniuschild) [reply src/config/db.config.js:1]: this `"env": {
    "amd": true
},` turns of this too.

**Review by parkerthegeniuschild — APPROVED** [review]

**Review by chiomadans1759 — APPROVED** [review]

File: src/config/passport.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (O-Osahon) [reply src/config/passport.js:1]: Thanks a lot bro

**Pull request closed without merging**